### PR TITLE
Refactor, delete person references

### PR DESCRIPTION
**Pull request by jellywaiyan**
Could not resolve error in: LogicmManagerTest lines 63 onwards

### CODE REVIEW

File: src/main/java/seedu/address/logic/parser/AddressBookParser.java [hunk 1]
```diff
@@ -102,27 +92,6 @@ public Command parseCommand(String userInput, Function<View, Boolean> viewVerifi
...
-            return new EditCommandParser().parse(arguments);
-
-        case DeleteCommand.COMMAND_WORD:
-            verifyView(viewVerifier, View.PERSONS);
-            return new DeleteCommandParser().parse(arguments);
-
-        case ClearCommand.COMMAND_WORD:
-            return new ClearCommand();
-
-        case FindCommand.COMMAND_WORD:
```
Comment (Badatprogrammiing) [inline src/main/java/seedu/address/logic/parser/AddressBookParser.java:120]: is this supposed to be deleted?

File: src/main/java/seedu/address/logic/parser/person/ParserUtil.java [hunk 2]
```diff
@@ -35,66 +31,6 @@ public static Index parseIndex(String oneBasedIndex) throws ParseException {
...
-     */
-    public static Email parseEmail(String email) throws ParseException {
-        requireNonNull(email);
-        String trimmedEmail = email.trim();
-        if (!Email.isValidEmail(trimmedEmail)) {
-            throw new ParseException(Email.MESSAGE_CONSTRAINTS);
-        }
-        return new Email(trimmedEmail);
-    }
-
```
Comment (Badatprogrammiing) [inline src/main/java/seedu/address/logic/parser/person/ParserUtil.java:97]: This entire file can de deleted right?

File: src/main/java/seedu/address/model/AddressBook.java [hunk 3]
```diff
@@ -269,12 +206,6 @@ public boolean equals(Object other) {
         }
 
         AddressBook otherAddressBook = (AddressBook) other;
-        return persons.equals(otherAddressBook.persons);
+        return assignments.equals(otherAddressBook.assignments);
```
Comment (Badatprogrammiing) [inline src/main/java/seedu/address/model/AddressBook.java:209]: Can u add internshipRoles.equals... and internshipTask.equals as well?

File: src/main/java/seedu/address/model/AddressBook.java [hunk 4]
```diff
@@ -80,65 +71,11 @@ public void setInternshipTasks(List<InternshipTask> internshipTasks) {
...
-    }
-
-    //// util methods
-
-    @Override
-    public String toString() {
-        return new ToStringBuilder(this)
-                .add("persons", persons)
-                .toString();
-    }
```
Comment (Badatprogrammiing) [inline src/main/java/seedu/address/model/AddressBook.java:134]: I dont think this toString method shld be removed, just update it with the current lists

File: src/main/java/seedu/address/model/AddressBook.java [hunk 5]
```diff
@@ -269,12 +206,6 @@ public boolean equals(Object other) {
...
 
         AddressBook otherAddressBook = (AddressBook) other;
-        return persons.equals(otherAddressBook.persons);
+        return assignments.equals(otherAddressBook.assignments);
     }
-
-    @Override
-    public int hashCode() {
-        return persons.hashCode();
-    }
```
Comment (Badatprogrammiing) [inline src/main/java/seedu/address/model/AddressBook.java:278]: same for this as wll, update with the current list

File: src/test/java/seedu/address/model/unique/UniqueListTest.java [hunk 6]
```diff
@@ -175,16 +150,6 @@ public void sets_nullList_throwsNullPointerException() {
...
-        List<UniqueMock> personList = Collections.singletonList(u3);
-        uniqueUniqueMockList.setList(personList);
-        UniqueList<UniqueMock> expectedUniqueUniqueMockList = new UniqueList<UniqueMock>();
-        expectedUniqueUniqueMockList.add(u3);
-        assertEquals(expectedUniqueUniqueMockList, uniqueUniqueMockList);
-    }
-
     @Test
     public void sets_listWithDuplicateUniqueMocks_throwsDuplicateUniqueMockException() {
         List<UniqueMock> listWithDuplicateUniqueMocks = Arrays.asList(u1, u1);
```
Comment (Badatprogrammiing) [inline src/test/java/seedu/address/model/unique/UniqueListTest.java:155]: This entire file, should not have test cases being deleted. I forgot to name to change all references to person

**Review by Badatprogrammiing — CHANGES_REQUESTED** [review]
as per the above comments

**Review by seewhyjay — APPROVED** [review]
sendit

**Review by Badatprogrammiing — APPROVED** [review]

**Pull request merged**